database/seeds/VouchersSeeder.php

```diff
@@ -32,26 +32,52 @@ public function run()
         $size = sizeOf($rvp_vouchers);
         // Assign the codes that match the paper.
         for ($i=60; $i<$size+60; $i++) {
-            $rvp_vouchers[$i-60]->code = 'RVNT123455'.$i;
-            $rvp_vouchers[$i-60]->sponsor_id = $rvp->id;
-            $rvp_vouchers[$i-60]->save();
+            $k = $i-60;
```

Comment:
Ah! simpler!

Reply:
Ha ha. Yeah - seems weird doing that calculation over and over. I think I can only blame myself though.